src/ToolBox/superpmi/superpmi/jitinstance.cpp

```diff
@@ -212,7 +212,8 @@ HRESULT JitInstance::StartUp(char*          PathToJit,
 
 bool JitInstance::reLoad(MethodContext* firstContext)
 {
-    FreeLibrary(hLib);
+    // TODO: Memory leaks here, but CILJit::ProcessShutdownWork is very expensive.
```

Comment:
We lose less then MB for each reload. I was not able to notice significant increase during replay, artificial test that just reloads jit many times shows that we don't lose much.

However, if you call ProcessShutdownWork , the replay time increases from 22 minutes to 26 on my machine.